cli/src/main/java/com/devonfw/tools/ide/tool/GlobalToolCommandlet.java

```diff
@@ -54,7 +54,7 @@ protected boolean runWithPackageManager(boolean silent, String... commandStrings
    * @return {@code true} if installation or uninstallation succeeds with any of the package manager commands, {@code false} otherwise.
    */
   protected boolean runWithPackageManager(boolean silent, List<PackageManagerCommand> pmCommands) {
-
+    
```

Comment:
check your IntelliJ settings?
Our formatter should remove such trailing whitespaces.
And the "Actions on Save" feature should be automatically enabled invoking the formatter and organize imports.
If that is not the case, we still struggle with the heritage of #52 and you should consult @jan-vcapgemini